test/unit/language-extensions/multi.spec.ts

```diff
@@ -24,6 +24,19 @@ test("multi example use case", () => {
         .expectToEqual({ a: "foo", b: 5 });
 });
 
+test("multi #995", () => {
```

Comment:
Our convention is to give the test a descriptive name, i.e `test("destructure LuaMultiReturn with rest"` and then add a comment with the PR link. (for example see https://github.com/TypeScriptToLua/TypeScriptToLua/blob/master/test/unit/builtins/object.spec.ts#L232 )